docs/.c4/model.c4

```diff
@@ -0,0 +1,596 @@
+
+      // external relationship
+      external-admin-loadbalancer -> istio-admin-ingress-gateway
+    }
+
+    external-services = infraServicesGroup {
+      title 'External Infrastructure Services'
+      summary ' '
+      description 'External infrastructure services supporting UDS Core components. These services run outside the Kubernetes cluster and provide persistent storage, databases, and other infrastructure dependencies required by UDS Core. Typically managed by cloud providers or infrastructure teams.'
+
+      ext-keycloak-db = infraService {
+        title 'External Keycloak Database'
+        summary ' '
+        description 'External PostgreSQL database storing Keycloak configuration and user data. Hosts Keycloak realms, clients, users, roles, and session information. Must be highly available and backed up regularly as it contains critical identity and access management data. Accessed via the Keycloak waypoint proxy over SSL/TLS on port 5432.'
+      }
+
+      ext-loki-storage = infraService {
+        title 'External Loki Storage'
+        summary ' '
+        description 'External object storage (S3-compatible) for Loki log data. Stores compressed log chunks and indexes for long-term retention. Supports AWS S3, MinIO, Azure Blob Storage, or other S3-compatible backends. Provides cost-effective, scalable storage for high-volume log data with configurable retention policies.'
+      }
+
+      ext-velero-storage = infraService {
+        title 'External Velero Storage'
+        summary ' '
+        description 'External object storage (S3-compatible) for Velero backups. Stores cluster backups including Kubernetes resource manifests and persistent volume snapshots. Critical for disaster recovery and cluster migration scenarios. Should be in a different failure domain than the cluster for true disaster recovery protection.'
+      }
+    }
+  }
+  // personas
+  admin = actor {
+    title 'Cluster Administrator'
+    summary ' '
+    description 'Administrator managing and monitoring the UDS Core platform. Platform operators and SREs who deploy, configure, and maintain UDS Core. Access administrative interfaces like Grafana for monitoring, Prometheus for metrics analysis, and Keycloak for identity management. Authenticate via Keycloak SSO with elevated privileges.'
+
+    // external relationship
+    admin -> external-admin-loadbalancer
+  }
+
+  enduser = actor {
+    title 'End User'
+    summary ' '
+    description 'End user accessing applications deployed on the UDS Core platform. Application users who access tenant workloads through the tenant ingress gateway. Authenticate via Keycloak SSO and benefit from UDS Core security features including mTLS encryption, network policies, and runtime security monitoring without requiring application-level changes.'
```

Comment:
```suggestion
    description 'End user accessing applications deployed on the UDS Core platform. Application users who access tenant workloads through the tenant ingress gateway.'
```

Feel like most of that second sentence is fluff/more about the applications than the user?